crates/bevy_asset/src/handle.rs

```diff
@@ -74,6 +74,13 @@ impl HandleId {
     }
 }
 
+/// A type for handler-related errors.
+#[derive(Debug)]
```

Comment:
```suggestion
#[derive(Debug, PartialEq)]
```

`PartialEq` is really nice on error types for tests, as it lets you check that the error is what you expected.